packages/site/src/components/Header.tsx

```diff
@@ -5,6 +5,7 @@ import { connectSnap, getThemePreference, getSnap } from '../utils';
 import { HeaderButtons } from './Buttons';
 import { SnapLogo } from './SnapLogo';
 import { Toggle } from './Toggle';
+import { version } from '../../package.json';
```

Comment:
This was producing a warning, Stack Overflow discussion here: https://stackoverflow.com/questions/64993118/

Reply:
Nice, thanks. Updated in [3d479dc](https://github.com/MetaMask/snap-simple-keyring/pull/71/commits/3d479dcee5e360b4b50d63706961baf19d3f3195) according to Stack Overflow top answer.